### ISSUE TITLE
editcontext: cursor not responding to keyboard events on a new file opened from file picker

### ISSUE THREAD

**Issue by deepak1556** [microsoft/vscode#228129]
* Select a file from the picker `Cmd + P`
* Use arrow keys to move the cursor
* Nothing happens
* Tab to previously opened files and refocus the new file then the cursor responds

**Comment by aiday-mar** [comment]
Hi @deepak1556 thank you for testing and filing this issue. I am wondering if after changing the setting you reloaded the window? In today's implementation the edit context was not automatically updated on configuration change, which causes some issues (I added a PR to remedy this https://github.com/microsoft/vscode/pull/228138).

**Comment by deepak1556** [comment]
Yup I did restart the application, attached a profile with screenshots when this issue happens.

[Trace-20240911T105937.json.zip](https://github.com/user-attachments/files/16956835/Trace-20240911T105937.json.zip)

**Comment by deepak1556** [comment]
Seems like there is source map issue with insiders, https://github.com/microsoft/vscode-internalbacklog/issues/5073

Here is the profile from OSS  [oss-electron-32.json.zip](https://github.com/user-attachments/files/16957486/oss-electron-32.json.zip), this is from `electron-31.x.y` branch which is based on Chromium 128. It could mean issue is either not related to the runtime or issue is still present in newer chromium versions.

**Comment by aiday-mar** [comment]
Thank you. Could I ask you if possible if you could include a screen recording of the issue, just so I can see what happens visually?

**Comment by aiday-mar** [comment]
I was able to reproduce the issue on yesterday's EditContext implementation. Today I have merged two fixes that tackle the incorrect focusing behavior of EditContext. I retried the code on the example I had previously used, and this issue seems to be fixed now. 

Two fixes:
https://github.com/microsoft/vscode/pull/228325
https://github.com/microsoft/vscode/pull/228348
